services/api/src/app.py

```diff
@@ -130,26 +133,28 @@ async def request(
     return response
 
 
-async def _blocking_response(response: ResponseModel):
-    logger.info(f"Responding to SID: `{response.session_id}`:")
-    response.log(logger)
+@sm.on("connect")
+async def connect(session_id: str, environ: Dict):
+    params = environ.get("QUERY_STRING")
+    params = dict(x.split("=") for x in params.split("&"))
+    
+    if "job_id" in params:
```

Comment:
I think `params.keys()` makes more sense. Also, what happens if `job_id` is not in params? Should it log an error or debug?